example/main.go

```diff
@@ -12,6 +12,11 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+// This example instruments a simple HTTP server-client application.
```

Comment:
```suggestion
// Package example instruments a simple HTTP server-client application.
```

Reply:
@MrAlias Are you sure? This is a main package

Reply:
Yep